config/routes.rb

```diff
@@ -1,15 +1,13 @@
 Rails.application.routes.draw do
   root "users#index"
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-end
-
-# Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-# Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-# Can be used by load balancers and uptime monitors to verify that the app is live.
-get "up" => "rails/health#show", as: :rails_health_check
+    resources :posts, only: [:index, :show, :create, :new] do
+      resources :comments, only: [:index, :show, :create]
```

Comment:
- [ ] As requested by the previous reviewers, kindly make sure that to create a comment for a post, only two routes are required ie ` resources :comments, only: [:new, :create]` Right now your app doesn't handle request from `http://localhost:3000/users/1/posts/1/comments/new` since you are missing this route, an action and a view template. This route is important because it fetch the page that let you to create a comment. For most web application, we are advise to use this route.
- [Optional] It will be nice to add links to create a post on the post index page so that the user won't do `posts/new` manually